mitmproxy/dump.py

```diff
@@ -158,7 +158,7 @@ def _readflow(self, paths):
 
     def add_event(self, e, level="info"):
         needed = dict(error=0, info=1, debug=2).get(level, 1)
-        if self.o.verbosity >= needed:
+        if self.o.verbosity or -1 >= needed:
```

Comment:
`verbosity` can be `None` which would result in comparing `NoneType` with `Int`. This used to work in Py2 but [not in Py3](https://docs.python.org/release/3.0/whatsnew/3.0.html#ordering-comparisons). So, This was the easiest 'fix' I could come up with.